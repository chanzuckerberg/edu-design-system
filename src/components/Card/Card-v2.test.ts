import { generateSnapshots } from '@chanzuckerberg/story-utils';
import type { StoryFile } from '@storybook/testing-react';

import * as stories from './Card-v2.stories';

describe('<Card /> (v2)', () => {
  generateSnapshots(stories as StoryFile);
});
