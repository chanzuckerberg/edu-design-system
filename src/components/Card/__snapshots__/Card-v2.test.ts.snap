// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Card /> (v2) BackgroundCallout story renders snapshot 1`] = `
<div
  class="p-8"
>
  <article
    class="card card--container-style-low card--container-color-call-out w-96"
  >
    <header
      class="card__header header--size-md"
    >
      <div
        class="header__text"
      >
        <div
          class="text text--overline-md header__eyebrow header--size-md"
        >
          Recommended for you
        </div>
        <div
          class="text text--headline-sm header__title header--size-md"
        >
          Question of the day
        </div>
        <div
          class="text text--body-lg header__sub-title header--size-md"
        >
          Get to know your colleagues
        </div>
      </div>
      <div
        class="header__action"
      >
        <div
          class="menu"
          data-headlessui-state=""
        >
          <button
            aria-expanded="false"
            aria-haspopup="menu"
            class=""
            data-headlessui-state=""
            id="headlessui-menu-button-:r8:"
            type="button"
          >
            <svg
              class="icon"
              fill="currentColor"
              height="2rem"
              role="img"
              style="--icon-size: 2rem;"
              viewBox="0 0 24 24"
              width="2rem"
              xmlns="http://www.w3.org/2000/svg"
            >
              <title>
                show more
              </title>
              <path
                d="M6 14C5.45 14 4.97933 13.804 4.588 13.412C4.196 13.0207 4 12.55 4 12C4 11.45 4.196 10.979 4.588 10.587C4.97933 10.1957 5.45 10 6 10C6.55 10 7.02067 10.1957 7.412 10.587C7.804 10.979 8 11.45 8 12C8 12.55 7.804 13.0207 7.412 13.412C7.02067 13.804 6.55 14 6 14ZM12 14C11.45 14 10.9793 13.804 10.588 13.412C10.196 13.0207 10 12.55 10 12C10 11.45 10.196 10.979 10.588 10.587C10.9793 10.1957 11.45 10 12 10C12.55 10 13.021 10.1957 13.413 10.587C13.8043 10.979 14 11.45 14 12C14 12.55 13.8043 13.0207 13.413 13.412C13.021 13.804 12.55 14 12 14ZM18 14C17.45 14 16.979 13.804 16.587 13.412C16.1957 13.0207 16 12.55 16 12C16 11.45 16.1957 10.979 16.587 10.587C16.979 10.1957 17.45 10 18 10C18.55 10 19.021 10.1957 19.413 10.587C19.8043 10.979 20 11.45 20 12C20 12.55 19.8043 13.0207 19.413 13.412C19.021 13.804 18.55 14 18 14Z"
              />
            </svg>
          </button>
        </div>
      </div>
    </header>
    <div
      class="card__body"
    >
      <div
        class="fpo my-4"
      >
        Card Body
      </div>
    </div>
    <footer
      class="card__footer"
    >
      <div
        class="button-group button-group--horizontal"
      >
        <button
          class="button button--layout-none button--primary button--lg button--variant-default"
          type="button"
        >
          <span
            class="button__text"
          >
            Primary Action
          </span>
        </button>
      </div>
    </footer>
    <div
      class="card__top-stripe top-stripe--none"
    />
  </article>
</div>
`;

exports[`<Card /> (v2) CancelMembership story renders snapshot 1`] = `
<div
  class="p-8"
>
  <article
    class="card card--container-style-low card--container-color-default w-96"
  >
    <header
      class="card__header header--size-sm"
    >
      <div
        class="header__text"
      >
        <div
          class="text text--title-sm header__title header--size-sm"
        >
          Cancel membership?
        </div>
        <div
          class="text text--body-sm header__sub-title header--size-sm"
        >
          We're sad to see you go
        </div>
      </div>
    </header>
    <div
      class="card__body"
    >
      <p
        class="text text--body-sm py-4"
      >
        Lorem ipsum dolor sit amet consectetur. Id pretium consequat consequat aliquam arcu
      </p>
    </div>
    <footer
      class="card__footer"
    >
      <div
        class="button-group button-group--horizontal"
      >
        <button
          class="button button--layout-none button--primary button--md button--variant-critical"
          type="button"
        >
          <span
            class="button__text"
          >
            Confirm cancellation
          </span>
        </button>
      </div>
    </footer>
    <div
      class="card__top-stripe top-stripe--none"
    />
  </article>
</div>
`;

exports[`<Card /> (v2) ChildCards story renders snapshot 1`] = `
<div
  class="p-8"
>
  <article
    class="card card--container-style-high card--container-color-call-out w-96"
  >
    <header
      class="card__header header--size-md"
    >
      <div
        class="header__text"
      >
        <div
          class="text text--headline-sm header__title header--size-md"
        >
          Card Group
        </div>
      </div>
    </header>
    <div
      class="card__body"
    >
      <article
        class="card card--container-style-high card--container-color-default child-card"
        draggable="true"
      >
        <header
          class="card__header header--size-md"
        >
          <div
            class="header__text"
          >
            <div
              class="text text--overline-md header__eyebrow header--size-md"
            >
              Eyebrow Text
            </div>
            <div
              class="text text--headline-sm header__title header--size-md"
            >
              Title text
            </div>
            <div
              class="text text--body-lg header__sub-title header--size-md"
            >
              Sub-title text
            </div>
          </div>
          <div
            class="header__action"
          >
            <div
              class="menu"
              data-headlessui-state=""
            >
              <button
                aria-expanded="false"
                aria-haspopup="menu"
                class=""
                data-headlessui-state=""
                id="headlessui-menu-button-:ra:"
                type="button"
              >
                <svg
                  class="icon"
                  fill="currentColor"
                  height="2rem"
                  role="img"
                  style="--icon-size: 2rem;"
                  viewBox="0 0 24 24"
                  width="2rem"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <title>
                    show more
                  </title>
                  <path
                    d="M6 14C5.45 14 4.97933 13.804 4.588 13.412C4.196 13.0207 4 12.55 4 12C4 11.45 4.196 10.979 4.588 10.587C4.97933 10.1957 5.45 10 6 10C6.55 10 7.02067 10.1957 7.412 10.587C7.804 10.979 8 11.45 8 12C8 12.55 7.804 13.0207 7.412 13.412C7.02067 13.804 6.55 14 6 14ZM12 14C11.45 14 10.9793 13.804 10.588 13.412C10.196 13.0207 10 12.55 10 12C10 11.45 10.196 10.979 10.588 10.587C10.9793 10.1957 11.45 10 12 10C12.55 10 13.021 10.1957 13.413 10.587C13.8043 10.979 14 11.45 14 12C14 12.55 13.8043 13.0207 13.413 13.412C13.021 13.804 12.55 14 12 14ZM18 14C17.45 14 16.979 13.804 16.587 13.412C16.1957 13.0207 16 12.55 16 12C16 11.45 16.1957 10.979 16.587 10.587C16.979 10.1957 17.45 10 18 10C18.55 10 19.021 10.1957 19.413 10.587C19.8043 10.979 20 11.45 20 12C20 12.55 19.8043 13.0207 19.413 13.412C19.021 13.804 18.55 14 18 14Z"
                  />
                </svg>
              </button>
            </div>
          </div>
        </header>
        <div
          class="card__top-stripe top-stripe--none"
        />
      </article>
    </div>
    <div
      class="card__top-stripe top-stripe--high"
    />
  </article>
</div>
`;

exports[`<Card /> (v2) CustomBrandCard story renders snapshot 1`] = `
<div
  class="p-8"
>
  <article
    class="card card--container-style-low card--container-color-custom-brand border-brand-red bg-brand-red w-96"
  >
    <header
      class="card__header header--size-md"
    >
      <div
        class="header__custom"
      >
        <div
          class="fpo w-full"
        >
          Card Header
        </div>
      </div>
    </header>
    <div
      class="card__body"
    >
      <div
        class="fpo"
      >
        Card Body
      </div>
    </div>
    <footer
      class="card__footer"
    >
      <div
        class="fpo"
      >
        Card Footer
      </div>
    </footer>
    <div
      class="card__top-stripe top-stripe--none"
    />
  </article>
</div>
`;

exports[`<Card /> (v2) CustomTopStripe story renders snapshot 1`] = `
<div
  class="p-8"
>
  <article
    class="card card--container-style-low card--container-color-default w-96"
  >
    <header
      class="card__header header--size-md"
    >
      <div
        class="header__text"
      >
        <div
          class="text text--overline-md header__eyebrow header--size-md"
        >
          Recommended for you
        </div>
        <div
          class="text text--headline-sm header__title header--size-md"
        >
          Question of the day
        </div>
        <div
          class="text text--body-lg header__sub-title header--size-md"
        >
          Get to know your colleagues
        </div>
      </div>
      <div
        class="header__action"
      >
        <div
          class="menu"
          data-headlessui-state=""
        >
          <button
            aria-expanded="false"
            aria-haspopup="menu"
            class=""
            data-headlessui-state=""
            id="headlessui-menu-button-:r6:"
            type="button"
          >
            <svg
              class="icon"
              fill="currentColor"
              height="2rem"
              role="img"
              style="--icon-size: 2rem;"
              viewBox="0 0 24 24"
              width="2rem"
              xmlns="http://www.w3.org/2000/svg"
            >
              <title>
                show more
              </title>
              <path
                d="M6 14C5.45 14 4.97933 13.804 4.588 13.412C4.196 13.0207 4 12.55 4 12C4 11.45 4.196 10.979 4.588 10.587C4.97933 10.1957 5.45 10 6 10C6.55 10 7.02067 10.1957 7.412 10.587C7.804 10.979 8 11.45 8 12C8 12.55 7.804 13.0207 7.412 13.412C7.02067 13.804 6.55 14 6 14ZM12 14C11.45 14 10.9793 13.804 10.588 13.412C10.196 13.0207 10 12.55 10 12C10 11.45 10.196 10.979 10.588 10.587C10.9793 10.1957 11.45 10 12 10C12.55 10 13.021 10.1957 13.413 10.587C13.8043 10.979 14 11.45 14 12C14 12.55 13.8043 13.0207 13.413 13.412C13.021 13.804 12.55 14 12 14ZM18 14C17.45 14 16.979 13.804 16.587 13.412C16.1957 13.0207 16 12.55 16 12C16 11.45 16.1957 10.979 16.587 10.587C16.979 10.1957 17.45 10 18 10C18.55 10 19.021 10.1957 19.413 10.587C19.8043 10.979 20 11.45 20 12C20 12.55 19.8043 13.0207 19.413 13.412C19.021 13.804 18.55 14 18 14Z"
              />
            </svg>
          </button>
        </div>
      </div>
    </header>
    <div
      class="card__body"
    >
      <div
        class="fpo my-4"
      >
        Card Body
      </div>
    </div>
    <footer
      class="card__footer"
    >
      <div
        class="button-group button-group--horizontal"
      >
        <button
          class="button button--layout-none button--primary button--lg button--variant-default"
          type="button"
        >
          <span
            class="button__text"
          >
            Primary Action
          </span>
        </button>
      </div>
    </footer>
    <div
      class="card__top-stripe top-stripe--high bg-brand-purple-lowEmphasis"
    />
  </article>
</div>
`;

exports[`<Card /> (v2) Default story renders snapshot 1`] = `
<div
  class="p-8"
>
  <article
    class="card card--container-style-low card--container-color-default w-96"
  >
    <header
      class="card__header header--size-md"
    >
      <div
        class="header__custom"
      >
        <div
          class="fpo w-full"
        >
          Card Header
        </div>
      </div>
    </header>
    <div
      class="card__body"
    >
      <div
        class="fpo"
      >
        Card Body
      </div>
    </div>
    <footer
      class="card__footer"
    >
      <div
        class="fpo"
      >
        Card Footer
      </div>
    </footer>
    <div
      class="card__top-stripe top-stripe--none"
    />
  </article>
</div>
`;

exports[`<Card /> (v2) IsInteractive story renders snapshot 1`] = `
<div
  class="p-8"
>
  <article
    class="card card--container-style-low card--container-color-default card--is-interactive w-96"
  >
    <header
      class="card__header header--size-md"
    >
      <div
        class="header__custom"
      >
        <div
          class="fpo w-full"
        >
          Card Header
        </div>
      </div>
    </header>
    <div
      class="card__body"
    >
      <div
        class="fpo"
      >
        Card Body
      </div>
    </div>
    <footer
      class="card__footer"
    >
      <div
        class="fpo"
      >
        Card Footer
      </div>
    </footer>
    <div
      class="card__top-stripe top-stripe--none"
    />
  </article>
</div>
`;

exports[`<Card /> (v2) TopStripe story renders snapshot 1`] = `
<div
  class="p-8"
>
  <article
    class="card card--container-style-low card--container-color-default w-96"
  >
    <header
      class="card__header header--size-md"
    >
      <div
        class="header__text"
      >
        <div
          class="text text--overline-md header__eyebrow header--size-md"
        >
          Recommended for you
        </div>
        <div
          class="text text--headline-sm header__title header--size-md"
        >
          Question of the day
        </div>
        <div
          class="text text--body-lg header__sub-title header--size-md"
        >
          Get to know your colleagues
        </div>
      </div>
      <div
        class="header__action"
      >
        <div
          class="menu"
          data-headlessui-state=""
        >
          <button
            aria-expanded="false"
            aria-haspopup="menu"
            class=""
            data-headlessui-state=""
            id="headlessui-menu-button-:r4:"
            type="button"
          >
            <svg
              class="icon"
              fill="currentColor"
              height="2rem"
              role="img"
              style="--icon-size: 2rem;"
              viewBox="0 0 24 24"
              width="2rem"
              xmlns="http://www.w3.org/2000/svg"
            >
              <title>
                show more
              </title>
              <path
                d="M6 14C5.45 14 4.97933 13.804 4.588 13.412C4.196 13.0207 4 12.55 4 12C4 11.45 4.196 10.979 4.588 10.587C4.97933 10.1957 5.45 10 6 10C6.55 10 7.02067 10.1957 7.412 10.587C7.804 10.979 8 11.45 8 12C8 12.55 7.804 13.0207 7.412 13.412C7.02067 13.804 6.55 14 6 14ZM12 14C11.45 14 10.9793 13.804 10.588 13.412C10.196 13.0207 10 12.55 10 12C10 11.45 10.196 10.979 10.588 10.587C10.9793 10.1957 11.45 10 12 10C12.55 10 13.021 10.1957 13.413 10.587C13.8043 10.979 14 11.45 14 12C14 12.55 13.8043 13.0207 13.413 13.412C13.021 13.804 12.55 14 12 14ZM18 14C17.45 14 16.979 13.804 16.587 13.412C16.1957 13.0207 16 12.55 16 12C16 11.45 16.1957 10.979 16.587 10.587C16.979 10.1957 17.45 10 18 10C18.55 10 19.021 10.1957 19.413 10.587C19.8043 10.979 20 11.45 20 12C20 12.55 19.8043 13.0207 19.413 13.412C19.021 13.804 18.55 14 18 14Z"
              />
            </svg>
          </button>
        </div>
      </div>
    </header>
    <div
      class="card__body"
    >
      <div
        class="fpo my-4"
      >
        Card Body
      </div>
    </div>
    <footer
      class="card__footer"
    >
      <div
        class="button-group button-group--horizontal"
      >
        <button
          class="button button--layout-none button--primary button--lg button--variant-default"
          type="button"
        >
          <span
            class="button__text"
          >
            Primary Action
          </span>
        </button>
      </div>
    </footer>
    <div
      class="card__top-stripe top-stripe--medium"
    />
  </article>
</div>
`;

exports[`<Card /> (v2) WhileDragging story renders snapshot 1`] = `
<div
  class="p-8"
>
  <article
    class="card card--container-style-low card--container-color-default card--is-dragging-true w-96"
  >
    <header
      class="card__header header--size-md"
    >
      <div
        class="header__custom"
      >
        <div
          class="fpo w-full"
        >
          Card Header
        </div>
      </div>
    </header>
    <div
      class="card__body"
    >
      <div
        class="fpo"
      >
        Card Body
      </div>
    </div>
    <footer
      class="card__footer"
    >
      <div
        class="fpo"
      >
        Card Footer
      </div>
    </footer>
    <div
      class="card__top-stripe top-stripe--none"
    />
  </article>
</div>
`;

exports[`<Card /> (v2) WithCustomizedHeader story renders snapshot 1`] = `
<div
  class="p-8"
>
  <article
    class="card card--container-style-low card--container-color-default w-96"
  >
    <header
      class="card__header header--size-md mb-size-2"
    >
      <div
        class="header__custom"
      >
        Displaying 
        <strong>
          some text
        </strong>
         with 
        <em>
          mixed formatting
        </em>
        .
      </div>
    </header>
    <div
      class="card__body"
    >
      <div
        class="fpo"
      >
        Card Body
      </div>
    </div>
    <footer
      class="card__footer"
    >
      <div
        class="fpo"
      >
        Card Footer
      </div>
    </footer>
    <div
      class="card__top-stripe top-stripe--none"
    />
  </article>
</div>
`;

exports[`<Card /> (v2) WithFullHeader story renders snapshot 1`] = `
<div
  class="p-8"
>
  <article
    class="card card--container-style-low card--container-color-default w-96"
  >
    <header
      class="card__header header--size-md"
    >
      <div
        class="header__text"
      >
        <div
          class="text text--overline-md header__eyebrow header--size-md"
        >
          Recommended for you
        </div>
        <div
          class="text text--headline-sm header__title header--size-md"
        >
          Question of the day
        </div>
        <div
          class="text text--body-lg header__sub-title header--size-md"
        >
          Get to know your colleagues
        </div>
      </div>
    </header>
    <div
      class="card__body"
    >
      <div
        class="fpo"
      >
        Card Body
      </div>
    </div>
    <footer
      class="card__footer"
    >
      <div
        class="fpo"
      >
        Card Footer
      </div>
    </footer>
    <div
      class="card__top-stripe top-stripe--none"
    />
  </article>
</div>
`;

exports[`<Card /> (v2) WithFullHeaderAndIcon story renders snapshot 1`] = `
<div
  class="p-8"
>
  <article
    class="card card--container-style-low card--container-color-default w-96"
  >
    <header
      class="card__header header--size-md"
    >
      <div
        class="header__icon"
      >
        <svg
          aria-hidden="true"
          class="icon"
          fill="currentColor"
          height="1.5rem"
          style="--icon-size: 1.5rem;"
          viewBox="0 0 24 24"
          width="1.5rem"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M5.85 17.1C6.7 16.45 7.65 15.9375 8.7 15.5625C9.75 15.1875 10.85 15 12 15C13.15 15 14.25 15.1875 15.3 15.5625C16.35 15.9375 17.3 16.45 18.15 17.1C18.7333 16.4167 19.1875 15.6417 19.5125 14.775C19.8375 13.9083 20 12.9833 20 12C20 9.78333 19.2208 7.89583 17.6625 6.3375C16.1042 4.77917 14.2167 4 12 4C9.78333 4 7.89583 4.77917 6.3375 6.3375C4.77917 7.89583 4 9.78333 4 12C4 12.9833 4.1625 13.9083 4.4875 14.775C4.8125 15.6417 5.26667 16.4167 5.85 17.1ZM12 13C11.0167 13 10.1875 12.6625 9.5125 11.9875C8.8375 11.3125 8.5 10.4833 8.5 9.5C8.5 8.51667 8.8375 7.6875 9.5125 7.0125C10.1875 6.3375 11.0167 6 12 6C12.9833 6 13.8125 6.3375 14.4875 7.0125C15.1625 7.6875 15.5 8.51667 15.5 9.5C15.5 10.4833 15.1625 11.3125 14.4875 11.9875C13.8125 12.6625 12.9833 13 12 13ZM12 22C10.6167 22 9.31667 21.7375 8.1 21.2125C6.88333 20.6875 5.825 19.975 4.925 19.075C4.025 18.175 3.3125 17.1167 2.7875 15.9C2.2625 14.6833 2 13.3833 2 12C2 10.6167 2.2625 9.31667 2.7875 8.1C3.3125 6.88333 4.025 5.825 4.925 4.925C5.825 4.025 6.88333 3.3125 8.1 2.7875C9.31667 2.2625 10.6167 2 12 2C13.3833 2 14.6833 2.2625 15.9 2.7875C17.1167 3.3125 18.175 4.025 19.075 4.925C19.975 5.825 20.6875 6.88333 21.2125 8.1C21.7375 9.31667 22 10.6167 22 12C22 13.3833 21.7375 14.6833 21.2125 15.9C20.6875 17.1167 19.975 18.175 19.075 19.075C18.175 19.975 17.1167 20.6875 15.9 21.2125C14.6833 21.7375 13.3833 22 12 22ZM12 20C12.8833 20 13.7167 19.8708 14.5 19.6125C15.2833 19.3542 16 18.9833 16.65 18.5C16 18.0167 15.2833 17.6458 14.5 17.3875C13.7167 17.1292 12.8833 17 12 17C11.1167 17 10.2833 17.1292 9.5 17.3875C8.71667 17.6458 8 18.0167 7.35 18.5C8 18.9833 8.71667 19.3542 9.5 19.6125C10.2833 19.8708 11.1167 20 12 20ZM12 11C12.4333 11 12.7917 10.8583 13.075 10.575C13.3583 10.2917 13.5 9.93333 13.5 9.5C13.5 9.06667 13.3583 8.70833 13.075 8.425C12.7917 8.14167 12.4333 8 12 8C11.5667 8 11.2083 8.14167 10.925 8.425C10.6417 8.70833 10.5 9.06667 10.5 9.5C10.5 9.93333 10.6417 10.2917 10.925 10.575C11.2083 10.8583 11.5667 11 12 11Z"
          />
        </svg>
      </div>
      <div
        class="header__text"
      >
        <div
          class="text text--headline-sm header__title header--size-md"
        >
          Question of the day
        </div>
        <div
          class="text text--body-lg header__sub-title header--size-md"
        >
          <span>
            Get to know your colleagues
          </span>
        </div>
      </div>
    </header>
    <div
      class="card__body"
    >
      <div
        class="fpo"
      >
        Card Body
      </div>
    </div>
    <footer
      class="card__footer"
    >
      <div
        class="fpo"
      >
        Card Footer
      </div>
    </footer>
    <div
      class="card__top-stripe top-stripe--none"
    />
  </article>
</div>
`;

exports[`<Card /> (v2) WithHorizontalPrimaryButton story renders snapshot 1`] = `
<div
  class="p-8"
>
  <article
    class="card card--container-style-low card--container-color-default w-96"
  >
    <header
      class="card__header header--size-md"
    >
      <div
        class="header__text"
      >
        <div
          class="text text--overline-md header__eyebrow header--size-md"
        >
          Recommended for you
        </div>
        <div
          class="text text--headline-sm header__title header--size-md"
        >
          Question of the day
        </div>
        <div
          class="text text--body-lg header__sub-title header--size-md"
        >
          Get to know your colleagues
        </div>
      </div>
      <div
        class="header__action"
      >
        <div
          class="menu"
          data-headlessui-state=""
        >
          <button
            aria-expanded="false"
            aria-haspopup="menu"
            class=""
            data-headlessui-state=""
            id="headlessui-menu-button-:r2:"
            type="button"
          >
            <svg
              class="icon"
              fill="currentColor"
              height="2rem"
              role="img"
              style="--icon-size: 2rem;"
              viewBox="0 0 24 24"
              width="2rem"
              xmlns="http://www.w3.org/2000/svg"
            >
              <title>
                show more
              </title>
              <path
                d="M6 14C5.45 14 4.97933 13.804 4.588 13.412C4.196 13.0207 4 12.55 4 12C4 11.45 4.196 10.979 4.588 10.587C4.97933 10.1957 5.45 10 6 10C6.55 10 7.02067 10.1957 7.412 10.587C7.804 10.979 8 11.45 8 12C8 12.55 7.804 13.0207 7.412 13.412C7.02067 13.804 6.55 14 6 14ZM12 14C11.45 14 10.9793 13.804 10.588 13.412C10.196 13.0207 10 12.55 10 12C10 11.45 10.196 10.979 10.588 10.587C10.9793 10.1957 11.45 10 12 10C12.55 10 13.021 10.1957 13.413 10.587C13.8043 10.979 14 11.45 14 12C14 12.55 13.8043 13.0207 13.413 13.412C13.021 13.804 12.55 14 12 14ZM18 14C17.45 14 16.979 13.804 16.587 13.412C16.1957 13.0207 16 12.55 16 12C16 11.45 16.1957 10.979 16.587 10.587C16.979 10.1957 17.45 10 18 10C18.55 10 19.021 10.1957 19.413 10.587C19.8043 10.979 20 11.45 20 12C20 12.55 19.8043 13.0207 19.413 13.412C19.021 13.804 18.55 14 18 14Z"
              />
            </svg>
          </button>
        </div>
      </div>
    </header>
    <div
      class="card__body"
    >
      <div
        class="fpo my-4"
      >
        Card Body
      </div>
    </div>
    <footer
      class="card__footer"
    >
      <div
        class="button-group button-group--horizontal"
      >
        <button
          class="button button--layout-none button--primary button--lg button--variant-default"
          type="button"
        >
          <span
            class="button__text"
          >
            Primary Action
          </span>
        </button>
      </div>
    </footer>
    <div
      class="card__top-stripe top-stripe--none"
    />
  </article>
</div>
`;

exports[`<Card /> (v2) WithSmallFullHeaderAndIcon story renders snapshot 1`] = `
<div
  class="p-8"
>
  <article
    class="card card--container-style-low card--container-color-default w-96"
  >
    <header
      class="card__header header--size-sm"
    >
      <div
        class="header__icon"
      >
        <svg
          aria-hidden="true"
          class="icon"
          fill="currentColor"
          height="1rem"
          style="--icon-size: 1rem;"
          viewBox="0 0 24 24"
          width="1rem"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M5.85 17.1C6.7 16.45 7.65 15.9375 8.7 15.5625C9.75 15.1875 10.85 15 12 15C13.15 15 14.25 15.1875 15.3 15.5625C16.35 15.9375 17.3 16.45 18.15 17.1C18.7333 16.4167 19.1875 15.6417 19.5125 14.775C19.8375 13.9083 20 12.9833 20 12C20 9.78333 19.2208 7.89583 17.6625 6.3375C16.1042 4.77917 14.2167 4 12 4C9.78333 4 7.89583 4.77917 6.3375 6.3375C4.77917 7.89583 4 9.78333 4 12C4 12.9833 4.1625 13.9083 4.4875 14.775C4.8125 15.6417 5.26667 16.4167 5.85 17.1ZM12 13C11.0167 13 10.1875 12.6625 9.5125 11.9875C8.8375 11.3125 8.5 10.4833 8.5 9.5C8.5 8.51667 8.8375 7.6875 9.5125 7.0125C10.1875 6.3375 11.0167 6 12 6C12.9833 6 13.8125 6.3375 14.4875 7.0125C15.1625 7.6875 15.5 8.51667 15.5 9.5C15.5 10.4833 15.1625 11.3125 14.4875 11.9875C13.8125 12.6625 12.9833 13 12 13ZM12 22C10.6167 22 9.31667 21.7375 8.1 21.2125C6.88333 20.6875 5.825 19.975 4.925 19.075C4.025 18.175 3.3125 17.1167 2.7875 15.9C2.2625 14.6833 2 13.3833 2 12C2 10.6167 2.2625 9.31667 2.7875 8.1C3.3125 6.88333 4.025 5.825 4.925 4.925C5.825 4.025 6.88333 3.3125 8.1 2.7875C9.31667 2.2625 10.6167 2 12 2C13.3833 2 14.6833 2.2625 15.9 2.7875C17.1167 3.3125 18.175 4.025 19.075 4.925C19.975 5.825 20.6875 6.88333 21.2125 8.1C21.7375 9.31667 22 10.6167 22 12C22 13.3833 21.7375 14.6833 21.2125 15.9C20.6875 17.1167 19.975 18.175 19.075 19.075C18.175 19.975 17.1167 20.6875 15.9 21.2125C14.6833 21.7375 13.3833 22 12 22ZM12 20C12.8833 20 13.7167 19.8708 14.5 19.6125C15.2833 19.3542 16 18.9833 16.65 18.5C16 18.0167 15.2833 17.6458 14.5 17.3875C13.7167 17.1292 12.8833 17 12 17C11.1167 17 10.2833 17.1292 9.5 17.3875C8.71667 17.6458 8 18.0167 7.35 18.5C8 18.9833 8.71667 19.3542 9.5 19.6125C10.2833 19.8708 11.1167 20 12 20ZM12 11C12.4333 11 12.7917 10.8583 13.075 10.575C13.3583 10.2917 13.5 9.93333 13.5 9.5C13.5 9.06667 13.3583 8.70833 13.075 8.425C12.7917 8.14167 12.4333 8 12 8C11.5667 8 11.2083 8.14167 10.925 8.425C10.6417 8.70833 10.5 9.06667 10.5 9.5C10.5 9.93333 10.6417 10.2917 10.925 10.575C11.2083 10.8583 11.5667 11 12 11Z"
          />
        </svg>
      </div>
      <div
        class="header__text"
      >
        <div
          class="text text--title-sm header__title header--size-sm"
        >
          Question of the day
        </div>
        <div
          class="text text--body-sm header__sub-title header--size-sm"
        >
          Get to know your colleagues
        </div>
      </div>
      <div
        class="header__action"
      >
        <div
          class="menu"
          data-headlessui-state=""
        >
          <button
            aria-expanded="false"
            aria-haspopup="menu"
            class=""
            data-headlessui-state=""
            id="headlessui-menu-button-:r0:"
            type="button"
          >
            <svg
              class="icon"
              fill="currentColor"
              height="2rem"
              role="img"
              style="--icon-size: 2rem;"
              viewBox="0 0 24 24"
              width="2rem"
              xmlns="http://www.w3.org/2000/svg"
            >
              <title>
                show more
              </title>
              <path
                d="M6 14C5.45 14 4.97933 13.804 4.588 13.412C4.196 13.0207 4 12.55 4 12C4 11.45 4.196 10.979 4.588 10.587C4.97933 10.1957 5.45 10 6 10C6.55 10 7.02067 10.1957 7.412 10.587C7.804 10.979 8 11.45 8 12C8 12.55 7.804 13.0207 7.412 13.412C7.02067 13.804 6.55 14 6 14ZM12 14C11.45 14 10.9793 13.804 10.588 13.412C10.196 13.0207 10 12.55 10 12C10 11.45 10.196 10.979 10.588 10.587C10.9793 10.1957 11.45 10 12 10C12.55 10 13.021 10.1957 13.413 10.587C13.8043 10.979 14 11.45 14 12C14 12.55 13.8043 13.0207 13.413 13.412C13.021 13.804 12.55 14 12 14ZM18 14C17.45 14 16.979 13.804 16.587 13.412C16.1957 13.0207 16 12.55 16 12C16 11.45 16.1957 10.979 16.587 10.587C16.979 10.1957 17.45 10 18 10C18.55 10 19.021 10.1957 19.413 10.587C19.8043 10.979 20 11.45 20 12C20 12.55 19.8043 13.0207 19.413 13.412C19.021 13.804 18.55 14 18 14Z"
              />
            </svg>
          </button>
        </div>
      </div>
    </header>
    <div
      class="card__body"
    >
      <div
        class="fpo"
      >
        Card Body
      </div>
    </div>
    <footer
      class="card__footer"
    >
      <div
        class="fpo"
      >
        Card Footer
      </div>
    </footer>
    <div
      class="card__top-stripe top-stripe--none"
    />
  </article>
</div>
`;
